Example/swipeable/AppleStyleSwipeableRow.js

```diff
@@ -44,13 +44,13 @@ export default class AppleStyleSwipeableRow extends Component {
       </Animated.View>
     );
   };
-  renderRightActions = progress => (
-    <View style={{ width: 192, flexDirection: 'row' }}>
-      {this.renderRightAction('More', '#C8C7CD', 192, progress)}
-      {this.renderRightAction('Flag', '#ffab00', 128, progress)}
-      {this.renderRightAction('More', '#dd2c00', 64, progress)}
-    </View>
-  );
+    renderRightActions = progress => (
+        <View style={{ width: 192, flexDirection: I18nManager.isRTL?'row-reverse':'row' }}>
+            {this.renderRightAction('More', '#C8C7CD', 192, progress)}
```

Comment:
Could you fix whitespaces?

Reply:
My IDE goes mad because of whitespace diffs - dome my best.